addons/binding/org.openhab.binding.samsungtv/src/main/java/org/openhab/binding/samsungtv/internal/handler/SamsungTvHandler.java

```diff
@@ -316,20 +252,26 @@ private synchronized void createService(RemoteDevice device) {
                     if (newService != null) {
                         if (existingService != null) {
                             stopService(existingService);
+                            startService(newService);
+                            logger.debug("Restarting service in UPnP mode for: {}, {} ({})", modelName, type, udn);
+                        } else {
+                            startService(newService);
+                            logger.debug("Started service for: {}, {} ({})", modelName, type, udn);
                         }
-
-                        startService(newService);
+                    } else {
+                        logger.trace("Skipping unknown UPnP service: {}, {} ({})", modelName, type, udn);
                     }
+
                 } else {
-                    logger.debug("Device rediscovered, clear caches");
+                    logger.debug("Service rediscoved, clearing caches: {}, {} ({})", modelName, type, udn);
                     existingService.clearCache();
                 }
                 putOnline();
             } else {
-                logger.debug("Ignore device={}", device);
+                // logger.trace("Ignore device={}", device);
```

Comment:
Should we remove it or leave it?

Reply:
I propose to remove entirely: pollutes the logs a lot and not very usefull.